lib/MusicBrainz/Server/Entity/URL/Encyclopedisque.pm

```diff
@@ -5,11 +5,7 @@ use Moose;
 extends 'MusicBrainz::Server::Entity::URL';
 with 'MusicBrainz::Server::Entity::URL::Sidebar';
 
-sub sidebar_name {
-    my $self = shift;
-
-    return "Encyclop\N{U+E9}disque";
-}
+sub sidebar_name { "Encyclop\N{U+E9}disque" }
```

Comment:
To keep simplifying, you may want to use `utf8` as in [Animationsong](https://github.com/metabrainz/musicbrainz-server/blob/v-2019-01-22/lib/MusicBrainz/Server/Entity/URL/Animationsong.pm#L4-L9) for example.